docs/data/toolpad/getting-started/overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+<p class="description">Toolpad is a low-code UI builder for creating apps with MUI's suite of component libraries.</p>
+
+## Introduction
+
+Toolpad is a self-hosted low-code internal tools builder designed to extend MUI's suite of React components.
+It's designed for developers of all trades who want to save time building internal applications.
+Drag and drop pre-built UI components, connect your data sources, release your app, and you're done!
+
+<img src="/static/toolpad/marketing/index-hero-video-poster.jpg" alt="Editor screenshot" width="960px" />
+
+## Advantages of Toolpad
+
+- **Build faster than ever before:** measure your development time in minutes rather than hours or days—simply drag and drop the components you need.
+- **Extensible with code:** build low-code with pro-code extensibility. You can write JavaScript anywhere in Toolpad as soon as you feel limited by the built-in features.
+- **Connect to any data source:** Toolpad lets you work with HTTP endpoints, Google Sheets, or PostgreSQL for backend functionality.
+- **Robust versioning and deployment:** use a Git workflow to seamlessly manage deployment and versioning of your app.
```

Comment:
> use a Git workflow to seamlessly manage deployment and versioning of your app.

* Not sure I understand this, we're not using git anywhere in the release workflow.
* We just removed versioning 2 weeks ago. The only thing you can do now is "deploy to production".